<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecept</class>
 <widget class="QDialog" name="AddRecept">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите имя:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>47</y>
     <width>139</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите категорию:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>12</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>87</y>
     <width>144</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время приготовления:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>127</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип блюда:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>167</y>
     <width>147</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить изобржение</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>125</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>45</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>244</y>
     <width>141</width>
     <height>159</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>202</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>203</y>
     <width>97</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>167</y>
     <width>25</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>244</y>
     <width>89</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Изображение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>379</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>244</y>
     <width>291</width>
     <height>159</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>46</y>
     <width>175</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить категорию</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>85</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>86</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить время </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>126</y>
     <width>174</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить тип блюда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>204</y>
     <width>122</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список продуктов:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>203</y>
     <width>159</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить продукт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
